webapp/components/emoji_picker/emoji_picker.jsx

```diff
@@ -0,0 +1,449 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+
+import $ from 'jquery';
+import * as Emoji from 'utils/emoji.jsx';
+import EmojiStore from 'stores/emoji_store.jsx';
+import PureRenderMixin from 'react-addons-pure-render-mixin';
+import ReactDOM from 'react-dom';
+import * as Utils from 'utils/utils.jsx';
+import ReactOutsideEvent from 'react-outside-event';
+
+import EmojiPickerCategory from './components/emoji_picker_category.jsx';
+import EmojiPickerItem from './components/emoji_picker_item.jsx';
+import EmojiPickerPreview from './components/emoji_picker_preview.jsx';
+import {injectIntl, intlShape, defineMessages, FormattedMessage} from 'react-intl';
+
+// This should include all the categories available in Emoji.CategoryNames
+const CATEGORIES = [
+    'recent',
+    'people',
+    'nature',
+    'food',
+    'activity',
+    'travel',
+    'objects',
+    'symbols',
+    'flags',
+    'custom'
+];
+
+const holders = defineMessages({
+    recent: {
+        id: 'emoji_picker.recent',
+        defaultMessage: 'Recently Used'
+    },
+    people: {
+        id: 'emoji_picker.people',
+        defaultMessage: 'People'
+    },
+    nature: {
+        id: 'emoji_picker.nature',
+        defaultMessage: 'Nature'
+    },
+    food: {
+        id: 'emoji_picker.food',
+        defaultMessage: 'Food'
+    },
+    activity: {
+        id: 'emoji_picker.activity',
+        defaultMessage: 'Activity'
+    },
+    travel: {
+        id: 'emoji_picker.travel',
+        defaultMessage: 'Travel'
+    },
+    symbols: {
+        id: 'emoji_picker.symbols',
+        defaultMessage: 'Symbols'
+    },
+    objects: {
+        id: 'emoji_picker.objects',
+        defaultMessage: 'Objects'
+    },
+    flags: {
+        id: 'emoji_picker.flags',
+        defaultMessage: 'Flags'
+    },
+    custom: {
+        id: 'emoji_picker.custom',
+        defaultMessage: 'Custom'
+    }
+
+});
```

Comment:
We shouldn't use injectIntl or intlShape or defineMessages anymore. Can you use `utils.localizeMessage` and/or `FormattedMessage` instead?